pandas/tests/test_categorical.py

```diff
@@ -4549,6 +4549,13 @@ def test_concat_categorical(self):
                             'h': [None] * 6 + cat_values})
         tm.assert_frame_equal(res, exp)
 
+    def test_rank_categorical(self):
+        exp = pd.Series([1., 2., 3., 4., 5., 6.], name='A')
```

Comment:
move this tests to ``pandas/tests/series/test_analytics.py`` (and put next to the other rank tests)